controller/client/model.go

```diff
@@ -2,7 +2,14 @@ package client
 
 import "github.com/filecoin-project/dealbot/tasks"
 
-type UpdateTaskRequest struct {
+type PopTaskRequest struct {
 	Status   tasks.Status `json:"status"`
 	WorkedBy string       `json:"worked_by,omitempty"` // which dealbot works on that task
 }
+
+type UpdateTaskRequest struct {
+	Status    tasks.Status `json:"status"`
+	StageName string
```

Comment:
I think a comment on what the fields are used for would be great.

I am guessing that `stageName` identifies a stage in task processing, and is not an actor's alias.  However, it is not clear how/if it relates to `Status`.